software/index.rst

```diff
@@ -0,0 +1,19 @@
+################################
+Software Available at the OLCF
+################################
+
+.. toctree::
+   :maxdepth: 2
+
+   ddl
+
+Additional Resources
```

Comment:
Looks like this was transplanted from the getting started guide. I don't think we need to repeat it here.